### DISCUSSION TITLE
Configuration

### DISCUSSION THREAD

**Discussion by ErwinAnwyhere365** [jensrossbach/node-red-contrib-chronos#80] [Q&A] [answered]
Hi all,

How can you actually make use of the timezone of that you have configured in the chronos-config? Right now, I have a situation where my server time is different and I would like to make use of a different time in the Time Switch node but with the message ingress it ignores the chornos configuration but it will take into account the server time.

Kind regards,
Erwin

**Comment by jensrossbach** [comment]
Hi,

I'm not sure if I understand your question. First of all, with "server" I assume you mean the host on which Node-RED runs, correct?

> How can you actually make use of the timezone of that you have configured in the chronos-config?

In the Chronos configuration, you do not configure a time zone, you only configure the geographical position for the calculation of the sun/moon times.

> Right now, I have a situation where my server time is different

Your server time is different... to what?

In general, all times in the Chronos nodes always relate to the time zone of the system on which Node-RED runs. There is currently no possibility to configure a different/custom time zone, this would be a new feature.


Regards,
Jens

**Reply by ErwinAnwyhere365** [reply]
Hi Jens, 

Let me clarify. The Node-Red instance runs on host which has a specific timezone. In the Chronos configuration I configure a location based on longitude and latitude. What I would expect is that based on the chronos configuration you are making your rules within the node. However, the rules follow the Node-Red instance time. According to what you are saying it will always make use of the Node Red host time so I need to raise a feature request. What I don't understand is why you even have a chronos configuration if in the end it always makes use of the Node-Red host time? Makes no sense to me.

Kind regards,
Erwin

**Reply by jensrossbach** [reply]
Yes, as written above, the location based on the longitude and latitude is solely used for the calculation of the sun/moon position, it has no influence on the time zone, which is instead always the same as the time zone configured on the host running Node-RED.

To be honest, I didn't consider the use case that your Node-RED host is in a different time zone than your local environment. But I agree that it might be a valid use case, therefore I propose that you create a feature request for adding the possibility of configuring the time zone in the configuration node.

Regards,
Jens

**Comment by ErwinAnwyhere365** [comment, marked as answer]
Thanks for the response James, I will request a feature request.

Kind regards,
Erwin
